packages/@aws-cdk/aws-codepipeline/test/integ.cfn-template-from-repo.lit.expected.json

```diff
@@ -322,4 +322,4 @@
       }
     }
   }
-}
+}
```

Comment:
Can you try to revert this change? I think the integ tests will pass without this ordering change.